packages/typography/src/styled/StyledFont.tsx

```diff
@@ -29,43 +29,46 @@ export const THEME_SIZES: Record<TypographySize, ThemeSize> = {
   '3xlarge': 'xxxl'
 };
 
-const fontStyles = (props: IStyledFontProps & ThemeProps<DefaultTheme>) => {
-  const monospace =
-    props.isMonospace && ['inherit', 'small', 'medium', 'large'].indexOf(props.size!) !== -1;
-  const fontFamily = monospace && props.theme.fonts.mono;
-  const direction = props.theme.rtl ? 'rtl' : 'ltr';
+const fontStyles = ({
+  hue,
+  isBold,
+  isMonospace,
+  size,
+  theme
+}: IStyledFontProps & ThemeProps<DefaultTheme>) => {
+  const monospace = isMonospace && ['inherit', 'small', 'medium', 'large'].indexOf(size!) !== -1;
+  const fontFamily = monospace && theme.fonts.mono;
+  const direction = theme.rtl ? 'rtl' : 'ltr';
   let fontSize;
   let fontWeight;
   let lineHeight;
-  let color;
+  let color = getColor({ theme, variable: 'foreground.default' });
```

Comment:
Sets default color correctly based on mode.

Reply:
@jzempel To maintain the current behavior, do we need to keep color as `undefined`? If so, the color would be inherited from the parent container. It will also be the user's responsibility to set the correct color based on the mode they're in.

Reply:
Yes, the current API structure needs to be kept (aka `undefined`)